OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -262,6 +262,9 @@ bool InverseKinematicsTool::run()
         else
             modelFromFile = false;
 
+        // although newly loaded model will be finalized
+        // there is no gaurantee that the _model has not been edited/modified
+        _model->finalizeFromProperties();
```

Comment:
True. But it is consistent with what [InverseDynamicsTool](https://github.com/opensim-org/opensim-core/blob/b4ca2ed24aa4f243871487c4007fb389e262d7d2/OpenSim/Tools/InverseDynamicsTool.cpp#L249) is doing. Also avoiding another commit/build cycle would be preferable, I think.